src/profile-logic/profile-data.js

```diff
@@ -937,8 +938,11 @@ export function getFriendlyThreadName(
         case 'plugin':
           label = 'Plugin';
           break;
+        default:
+        // should we throw here ?
```

Comment:
In general I think I'm in favor of throwing aggressively for failed assumptions.